demo/app/app.py

```diff
@@ -11,57 +13,41 @@
 from dash import html
 import dash_bootstrap_components as dbc
 
-from Predictor import XGBoostPredictor
-from Predictor import LSTMPredictor
-from Prescriptor import Prescriptor
-from constants import ALL_LAND_USE_COLS
-from constants import CHART_COLS
-from constants import CONTEXT_COLUMNS
-from constants import ALL_DIFF_LAND_USE_COLS
-from constants import LAND_USE_COLS
-from constants import PRESCRIPTOR_LIST
-from constants import DEFAULT_PRESCRIPTOR_IDX
-from constants import PREDICTOR_LIST
-from constants import SLIDER_PRECISION
-from constants import MAP_COORDINATE_DICT
-from constants import CO2_JFK_GVA
-from constants import HISTORY_SIZE
-from constants import PRESCRIPTOR_COLS
-from constants import CO2_PERSON
-from constants import CHART_TYPES
-from constants import PARETO_CSV_PATH
-from utils import add_nonland
-from utils import create_map
-from utils import create_check_options
-from utils import compute_percent_change
-from utils import create_treemap
-from utils import create_pie
-from utils import create_pareto
+from . import Predictor
+from . import Prescriptor
+from . import constants
+from . import utils
 
 app = Dash(__name__,
            external_stylesheets=[dbc.themes.BOOTSTRAP, dbc.icons.BOOTSTRAP],
            prevent_initial_callbacks="initial_duplicate")
+server = app.server
 
 # TODO: should we load all our data into a store?
 # This seems more secure.
-df = pd.read_csv("../data/gcb/processed/gb_br_ch_eluc.csv", index_col=0)
+df = pd.read_csv(constants.DATA_FILE_PATH, index_col=constants.INDEX_COLS)
```

Comment:
Added a time/lat/lon index to the dataframe so that we are able to eliminate the context store and just grab it from the df directly.